README.adoc

```diff
@@ -54,6 +54,34 @@ return a `200 OK` response that contains all of the JVM's system properties.
 [role="command"]
 include::{common-includes}/gitclone.adoc[]
 
+=== Try what you'll build
+To try out the application, first navigate to the `finish` directory and then run the following Maven goals to build the application and run it inside Open Liberty:
```

Comment:
Use common adoc where applicable.

Reply:
We're keeping the commands separate for this intro guide so this one isn't applicable.